binder/environment.yml

```
@@ -1,10 +1,11 @@
 name: dask-tutorial
 channels:
   - conda-forge
+  - anaconda
 dependencies:
   - python=3.7
   - numpy=1.16.4
-  - nomkl
+  - anaconda:nomkl
```

This probably needs 2 `:`s.

```suggestion
  - anaconda::nomkl
```

---

Also is there a reason we need `nomkl`?